markov.js

```diff
@@ -74,16 +106,24 @@ class MarkovGenerator {
     }
 
     generate(corpusName, length) {
-        if (!this.corpora[corpusName]) return "That is not a thing (no such corpus).";
+        if (!this.corpora[corpusName])
+            return 'That is not a thing (no such corpus).';
 
-        let corpus = this.corpora[corpusName];
+        const corpus = this.corpora[corpusName];
+        const sentences = [];
```

Comment:
Assign once. An array is an object, so manipulation is only properties of the object (hence you aren't actually changing the object itself)